data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,458 @@
+}
+
+obj_tools = {
+   name = "obj_tools",
+   title = _ "Build tools quickly",
+   number = 1,
+   body = objective_text(_"Make new tools",
+      p(_[[Most of our tools have been lost in the ship tragedy. Our people will not be able to fulfill their duties without proper tools. Luckily we still have some iron ore and plenty of coal. Build a production of tools, it is a prerequisite for success on island. Don’t forget to build a smelter as well.]]))
+}
+
+obj_mining = {
+   name = "obj_mining",
+   title = _ "Explore the mountains for ressources",
+   number = 3,
+   body = objective_text(_"Build Mines",
+      p(_[[As we are able to make tools again, our supply of mining ressources needs to be replenished soon. Explore the mountains and build at least one mine of each type to mine coal, iron and gold.]]))
+}
+
+obj_explore = {
+   name = "obj_explore",
+   title = _"Explore the surroundings",
+   number = 1,
+   body = objective_text(_"Explore your surroundings",
+   p(_[[We are pretty sure that we are not alone on the island. We do not know if the others on this island are friends or foes so we need to treat carefully. Expand and explore the island by building at least one habitat for a scout and military buildings. Build at least one tower for his bigger view range.]]))
+}
+
+obj_spidercloth = {
+   name = "obj_spidercloth_production",
+   title = _"Build a spider farm and a weaving mill",
+   number = 2,
+   body = objective_text(_"Spidercloth Production",
+      p(_[[We need some spidercloth urgently as we are running out of it. So it is essential to spare every piece we still have. Reduce the input of all buildings that consume spidercloth. Don’t forget the constructionsites.]]) ..
+      p(_[[After that we need to have a farm under all circumstances as it needs spidercloth to be build. Additionally we need a spiderfarm and a weaving mill. Later on a gold spinning mill would be helpful as well.]]))
+}
+
+obj_trading_post = {
+   name = "obj_trading_post",
+   title = _"Build a trading post",
+   number = 1,
+   body = objective_text(_"Upgrade our scout’s house to a trading post",
+   p(_[[To be able to pay the tributes to Kalitath, and keep contact with Jundlina. We need to enhance our scout’s house to a trading post.]]))
+}
+
+obj_tribute = {
+   name = "obj_tribute",
+   title = _"Pay all tributes to Kalitath",
+   number = 1,
+   body = objective_text(_"Send our trader to Kalitath to deliver the demanded goods",
+   p(_[[We need to send the wares to Kalitath with our trader. He will pick them up as soon as a batch is ready in his trading post.]]) ..
+   p(_[[The following batches need to be delivered:]]) ..
+   li(_[[Five batches with five logs.]]) ..
+   li(_[[Four batches with four coal.]]) ..
+   li(_[[Three batches with three iron.]]) ..
+   li(_[[Two batches with two gold.]]))
```

Comment:
```suggestion
   li(_[[Five batches of five logs each.]]) ..
   li(_[[Four batches of four lumps of coal each.]]) ..
   li(_[[Three batches of three iron bars each.]]) ..
   li(_[[Two batches of two gold bars each.]]))
```
Sane for the second batch list below